cleanlab/count.py

```diff
@@ -113,15 +113,21 @@ class 0, 1, ..., K-1. `pred_probs` should have been computed using 3 (or
 
     if confident_joint is None:
         # Original non-calibrated counts of confidently correctly and incorrectly labeled examples.
-        confident_joint = compute_confident_joint(
-            labels=labels, pred_probs=pred_probs, calibrate=False
+        computed_confident_joint = compute_confident_joint(
+            labels=labels,
+            pred_probs=pred_probs,
+            calibrate=False,
         )
+    else:
+        computed_confident_joint = confident_joint.copy()
```

Comment:
is the `.copy()` truly necessary here?

Reply:
Nope! I will remove but I try an avoid all direct assignments without it

Reply:
cool just wanted to make sure my understanding of mypy is not wrong :D